test/user.test.js

```diff
@@ -205,6 +205,73 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done(err);
+            assert.equal(err.statusCode, 422);
+            assert(false, 'Error should have been thrown');
+          });
+        } catch (e) {
+          done();
+        }
+      });
+
+      it('accepts passwords that are exactly 72 characters long', function(done) {
+        User.create({ email: 'b@c.com', password: pass72 }, function(err, user) {
+          if (err) return done(err);
+          assert(user.id);
+          assert(!err);
+
+          done();
+        });
+      });
+
+      it('allows login with password exactly 72 characters long', function(done) {
+        User.create({ email: 'b@c.com', password: pass72 }, function(err) {
+          User.login({ email: 'b@c.com', password: pass72 }, function(err, accessToken) {
+            if (err) return done(err);
+            assertGoodToken(accessToken);
+            assert(accessToken.id);
+            assert(!err);
+
+            done();
+          });
+        });
+      });
+
+      it('rejects login when password is proper pass of 72 chars + extra added chars',
```

Comment:
I meant the sentence for the description needs improvement to clarify "72 chars + extra added chars" means to someone who doesn't know the situation. Either add a comment or improve the description, I vote for the latter.